Source/JavaScriptCore/tools/JSDollarVM.cpp

```diff
@@ -2847,7 +2847,7 @@ JSC_DEFINE_HOST_FUNCTION(functionVMTaintedState, (JSGlobalObject* globalObject,
     DollarVMAssertScope assertScope;
     VM& vm = globalObject->vm();
 
-    SourceTaintedOrigin sourceTaintedOrigin = sourceTaintedOriginFromStack(vm, callFrame);
+    auto [sourceTaintedOrigin, url] = sourceTaintedOriginFromStack(vm, callFrame);
```

Comment:
We don't get an unused variable error for `url`?

n.b. In C++26 we should get the syntactic sugar to ignore `url` -- https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2020/p2169r0.pdf

Reply:
Oh, apparently clang doesn't warn about unused variables as long as you use at least one of the results from the structured bindings assignment. I think I can make this robust if this ever changes, though, by just using `.first`:

```
auto sourceTaintedOrigin = sourceTaintedOriginFromStack(vm, callFrame).first;
```

Reply:
It’s probably because structured binding “variables” aren’t actually variables and is why you can’t capture them in lambdas (in some cases I forget which though)